app/views/filters/index.html.erb

```diff
@@ -1,3 +1,48 @@
 <%= turbo_frame_tag 'frame_filters' do %>
-  <p>You have a total of <%= keywords_count %> keywords.</p>
+  <p>
+    <%= t('filters.total_of_keywords', keywords_count: keywords_count) %>
+  </p>
+  <%= form_with url: keywords_path, method: :get, class: 'form-filter', data: { turbo: true, turbo_frame: 'frame_list_keywords' } do |form| %>
```

Comment:
@malparty How about using something like `form_for filter`, where filter is using the model we created in backend PR. Going this way, the values will be prefilled in the form.

Reply:
Yes, it would make sense (even if today we do not fill any value). 
I will update that once I will understand the root cause for system test failure ;-)
Thanks @ankitkalia1195  :)

Reply:
@ankitkalia1195 @malparty Actually `form_with` is a new thing that works both(model and non model), its combination of `form_for` and `form_tag`.  And `form_for` and `form_tag` bother will be removed future in favor of `form_with`.

So if needed we can pass the model with `form_with` like

```erb
<%= form_with model: your_object do |form| %>
...
...
<% end %>
```

More detail https://m.patrikonrails.com/rails-5-1s-form-with-vs-old-form-helpers-3a5f72a8c78a, https://stackoverflow.com/a/58112003

Reply:
After re-reading this, as I'm already using the form_with tag, I think I will implement de model change only in the nice-to-haves tasks:
- #87 
- #88 

Let me know if that is ok or not @ankitkalia1195  @junan  :) 
Thanks,

Reply:
Sure, its fine for now @malparty